source/vibe/web/rest.d

```diff
@@ -1,12 +1,228 @@
 /**
-	Automatic REST interface and client code generation facilities.
-
-	Copyright: © 2012-2013 RejectedSoftware e.K.
-	License: Subject to the terms of the MIT license, as written in the included LICENSE.txt file.
-	Authors: Sönke Ludwig, Михаил Страшун
-*/
+ * Module to work with REST API, from both server and client side.
+ *
+ * This modules aims to provide a typesafe way to deal with REST API.
+ * As they are quite a lot of similarities between them, this module use
+ * D interfaces as the way to represent REST API.
+ * While REST is not HTTP specific, the vast majority of use cases rely on it,
+ * and so does this module.
+ *
+ * This module, while being usable and more convenient from both client and
+ * server side, is also usable solely for its client or server implementation.
```

Comment:
The point is that the server and client part should be able to be used independently, so you could write an interface that allows you to query a server which runs Java. That was the motivation behind the various `@XXXXparams`. Does it makes it clearer ?